ts/src/coinlist.ts

```diff
@@ -0,0 +1,2246 @@
+        //
+        //     {
+        //         message: 'Order cancellation request received.',
+        //         timestamp: '2023-10-26T10:29:28.652Z'
+        //     }
+        //
+        const orders = [ response ];
+        return this.parseOrders (orders, market);
+    }
+
+    async cancelOrder (id: string, symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#cancelOrder
+         * @description cancels an open order
+         * @param {string} id order id
+         * @param {string} symbol not used by coinlist cancelOrder ()
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} An [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        const request = {
+            'order_id': id,
+        };
+        const response = await this.privateDeleteV1OrdersOrderId (this.extend (request, params));
+        //
+        //     {
+        //         message: 'Cancel order request received.',
+        //         order_id: 'd36e7588-6525-485c-b768-8ad8b3f745f9',
+        //         timestamp: '2023-10-26T14:36:37.559Z'
+        //     }
+        //
+        return this.parseOrder (response);
+    }
+
+    async cancelOrders (ids, symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#cancelOrders
+         * @description cancel multiple orders
+         * @param {string[]} ids order ids
+         * @param {string} symbol not used by coinlist cancelOrders ()
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} an list of [order structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        params = ids;
+        const response = await this.privateDeleteV1OrdersBulk (params);
+        return response;
+    }
+
+    async createOrder (symbol: string, type: OrderType, side: OrderSide, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#createOrder
+         * @description create a trade order
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit' or 'stop_market' or 'stop_limit' or 'take_market' or 'take_limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} an [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
```

Comment:
we need to add the `postOnly`, and `reduceOnly` features to the docs (params.postOnly ...) you can check other implementations